.ci/linux-steps.yaml

```diff
@@ -52,7 +52,7 @@ steps:
       export GOPATH=$PWD/src/mlpack/bindings/go
       go get -u -t gonum.org/v1/gonum/...
     fi
-    cmake $(CMakeArgs) -DPYTHON_EXECUTABLE=`which python` ..
+    cmake $(CMakeArgs) -DPYTHON_EXECUTABLE=`which python` -DCEREAL_INCLUDE_DIR=$PWD/cereal-1.3.0/include/ ..
```

Comment:
Are you sure `$PWD` is right here?  It looks like you are in `build/`, so maybe `../cereal-1.3.0/include/`?

Reply:
Thanks, I did not know where I am,

Reply:
I'm just guessing, I'm not totally sure.  If that fails too just try doing `ls` and other commands in the build to try and figure out what is going on. :)